force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -163,9 +183,12 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer generateRandomNumber() {
         // Use Math class to generate number between 50 and 100
-
+        Integer min = 50;
+        Integer max = 100;
+        Double randomDouble = Math.floor(Math.random() * (max - min +1)) + min;
+        Integer randomInteger = randomDouble.intValue();
```

Comment:
Here we can save time by declaring an Integer using casting (which will also round off the number in the process):
```suggestion
        Integer randomInteger = (Integer) (Math.random() * (max - min + 1) + min;
```